integration/presets/tanstack.ts

```diff
@@ -0,0 +1,33 @@
+import { applicationConfig } from '../models/applicationConfig.js';
+import { templates } from '../templates/index.js';
+
+const clerkTanStackLocal = `file:${process.cwd()}/packages/tanstack-start`;
+
+const router = applicationConfig()
+  .setName('tanstack-router')
+  .useTemplate(templates['tanstack-router'])
+  .setEnvFormatter('public', key => `VITE_${key}`)
+  // `--install-links` ensures that the @clerk/tanstack-start is installed as a regular dependancie instead of creating symlink
+  // please do not remove this flag as it's going to break the setup
+  .addScript('setup', 'npm i --install-links')
```

Comment:
Oh wow this line actually fixed the issue we're having [here](https://linear.app/clerk/issue/ECO-17/explore-our-options-for-csr-only-support-in-astro#comment-b8827a6e)